content/2_Bronze/Intro_Sets.mdx

```diff
@@ -619,28 +662,30 @@ Collection. See the following code snippet for an example (it works equivalently
 to the code above):
 
 ```java
-Set<Integer> s = new TreeSet<>();
-// s starts as {0, 1, 2}
-s.add(0); s.add(1); s.add(2);
-
-Set<Integer> toRemove = new TreeSet<>();
-for (Integer a : s) {
-	if (a == 0 || a == 2) {
-		toRemove.add(a);
+Map<Integer, Integer> s = new TreeMap<>();
+// m starts as {0: 0, 1: 1, 2: 2}
+m.put(0, 0); m.put(1, 1); m.put(2, 2);
+
+Set<Integer> keysToRemove = new TreeSet<>();
+for (Map.Entry<Integer, Integer> entry : m.entrySet()) {
+	int key = entry.getKey();
+	if (key == 0 || key == 2) {
+		keysToRemove.add(key);
 	}
 }
 
-s.removeAll(toRemove);
+// removes all entries with key in keysToRemove
+m.keySet().removeAll(keysToRemove);
 
 // s ends as {1}
-System.out.println("Entries:");
-for (Integer a : s) {
-	System.out.println(a);
+System.out.println("Remaining Entries:");
+for (Map.Entry<Integer, Integer> entry : m.entrySet()) {
+	System.out.println(entry.getKey() + " " + entry.getValue());
 }
 
 /*
 Entries:
-1
+1 1
```

Comment:
put as single line comment maybe

Reply:
i think the point is to show what the code will output

Reply:
if so then at least format the multiline acceptably
```cpp
/*
 * for reference
 * something like this
 */
```

Reply:
wait you alr did it, sorry